README.md

```diff
@@ -123,7 +113,9 @@ CGO_ENABLED=1 go build -o bin/ -tags='testnet netgo timetzdata' -trimpath -a -ld
 `hostd` includes a `Dockerfile` which can be used for building and running
 hostd within a docker container. The image can also be pulled from `ghcr.io/siafoundation/hostd`.
 
-## Running container
+## Mainnet
```

Comment:
Not sure if #198 messed this up or not, but this feels out of order. I would be okay with renaming `### Production` to `### Mainnet`